Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

```diff
@@ -60,6 +60,17 @@ Once you've assigned a Hostname to your Umbraco Cloud environment, you may want
 All hostnames added to an Umbraco Cloud project's environment will get a TLS (HTTPS) certificate added, by default. The certificate is issued by Cloudflare and valid for 1 year after which it will be automatically renewed. Everything around certificates and renewals is handled for you and you only need to make sure that the DNS records are configured according to our recommendations listed above.
 
 You will need to **remove the old DNS entry** before the Cloudflare service generates a new certificate for your Hostname.
+:::note
+Please note the following upcoming change in Certificate Authority (CA) used to issue certificates for all Umbraco Cloud sites' custom hostnames.
+
+**Certificates for new custom hostnames:**
+On September 26th 2022 certificates will be issued using  'Google Trust Services' instead of 'DigiCert', and Certificate validity will be decreased from 1 year to 90 days.
+
+**Certificates for existing custom hostnames:**
+From October 31st Certificate renewals will no longer use 'DigiCert' as the issuing CA. The renewed certificate will instead by issued by 'Google Trust Services',  and Certificate validity will be decreased from 1 year to 90 days.
+
+No action is required unless you set a Certificate Authority Authorization (CAA) record on your domain.
```

Comment:
```suggestion
Please note the following upcoming change in Certificate Authority (CA) used to issue certificates for all Umbraco Cloud sites' custom hostnames.

**Certificates for new custom hostnames:**
On September 26th, 2022 certificates will be issued using  'Google Trust Services' instead of 'DigiCert', and Certificate validity will be decreased from 1 year to 90 days.

**Certificates for existing custom hostnames:**
From October 31st Certificate renewals will no longer use 'DigiCert' as the issuing CA. The renewed certificate will instead be issued by 'Google Trust Services',  and Certificate validity will be decreased from 1 year to 90 days.

No action is required unless you set a Certificate Authority Authorization (CAA) record on your domain.
```